app.py

```diff
@@ -1,72 +1,64 @@
 #!/usr/bin/env python3
 
-import json
 import os
-import cherrypy
-from elasticsearch import Elasticsearch
-from lib.model.part import Part
-from lib.render import PHRender
+from pathlib import Path
 
+from elasticsearch import AsyncElasticsearch
+from quart import Quart, redirect, request, render_template, safe_join, send_file
 
-class Search:
-    def __init__(self):
-        self.phrender = PHRender()
+from lib.model.part import Part
 
-        self.env = self.phrender.env
-        self.template = self.phrender.get_class_template(self.__class__)
 
-        if 'ELASTICSEARCH' in os.environ:
-            self.es = Elasticsearch([os.environ['ELASTICSEARCH']])
-        else:
-            self.es = Elasticsearch()
+app = Quart(__name__)
 
-    def render(self, page=None):
-        """Render the template associated with the page.
+STATIC_DIR = Path(__file__).parent / '_site'
 
-        If the client requests JSON, then this returns a JSON dump of the
-        """
+@app.route("/", defaults={'path': ""})
+@app.route("/<path:path>")
+async def serve(path: str):
+    p = safe_join(STATIC_DIR, path)
 
-        if page is None:
-            page = {}
+    if p.is_dir():
+        p/= 'index.html'
```

Comment:
Huh, that's new to me.

A little surprised that the linter didn't say something about a lack of space.

Reply:
Good point, re: space, and fixed.
Regarding there being an assignment operator, I didn't really think about it, just assumed `/=` does the same as `x = x / y` in the absence of an explicit implementation of `__itruediv__`.